lib/features/sending_queue/presentation/sending_queue_view.dart

```diff
@@ -23,31 +31,86 @@ class SendingQueueView extends GetWidget<SendingQueueController> with AppLoaderM
+              _buildListButtonSelectionForMobile(context),
             ]
           ),
         ),
       ),
-      floatingActionButton: ComposeFloatingButton(
-        scrollController: controller.listSendingEmailController,
-        onTap: () => controller.dashboardController!.goToComposer(ComposerArguments())
-      ),
+      floatingActionButton: _buildFloatingButtonCompose(context)
     );
   }
 
+  Widget _buildFloatingButtonCompose(BuildContext context) {
+    return Obx(() {
+      return Container(
+        padding: controller.listEmailSelected.isNotEmpty ? const EdgeInsets.only(bottom: 70) : EdgeInsets.zero,
+        child: ComposeFloatingButton(
+            scrollController: controller.listSendingEmailController,
+            onTap: () => controller.dashboardController!.goToComposer(ComposerArguments())
+        ),
+      );
+    });
+  }
+
   Widget _buildListSendingEmails(BuildContext context) {
     return Obx(() {
       if (controller.dashboardController!.listSendingEmails.isNotEmpty) {
-        return ListView.builder(
-          controller: controller.listSendingEmailController,
-          padding: EdgeInsets.zero,
-          itemCount: controller.dashboardController!.listSendingEmails.length,
-          itemBuilder: (context, index) => SendingEmailTileWidget(
-            sendingEmail: controller.dashboardController!.listSendingEmails[index]
-          )
+        return Focus(
+          focusNode: controller.focusNodeKeyBoard,
+          child: ListView.builder(
+            controller: controller.listSendingEmailController,
+            padding: EdgeInsets.zero,
+            itemCount: controller.dashboardController!.listSendingEmails.length,
+            itemBuilder: (context, index) {
+              return Obx(() => SendingEmailTileWidget(
+                selectModeAll: controller.currentSelectMode.value,
+                sendingEmail: controller.dashboardController!.sendingEmailsInCurrentMailbox[index],
+                onLongPress: () => controller.pressEmailAction(
+                  context,
+                  EmailActionType.selection,
+                  controller.dashboardController!.sendingEmailsInCurrentMailbox[index]),
+                onLeading: () => controller.pressEmailAction(
+                  context,
+                  EmailActionType.selection,
+                  controller.dashboardController!.sendingEmailsInCurrentMailbox[index]),
+                onTap: () => controller.pressEmailAction(
+                  context,
+                  EmailActionType.edit,
+                  controller.dashboardController!.sendingEmailsInCurrentMailbox[index]),
+              ));
+            }
+          ),
+        );
+      } else {
+        return const SizedBox.shrink();
+      }
+    });
+  }
+
+  Widget _buildListButtonSelectionForMobile(BuildContext context) {
+    return Obx(() {
+      if ((!BuildUtils.isWeb || (BuildUtils.isWeb && controller.isSelectionEnabled()
```

Comment:
SendingQueueView only displayed on` mobile` so you don't need to check `isWeb` condition